AutomatedTesting/Gem/PythonTests/Prefab/tests/duplicate_prefab/DuplicatePrefab_ContainingASingleEntity.py

```diff
@@ -28,6 +32,20 @@ def DuplicatePrefab_ContainingASingleEntity():
     # Duplicates the prefab instance
     Prefab.duplicate_prefabs([car])
 
+    # Test undo/redo on prefab duplication
+    general.undo()
+    wait_for_propagation()
+    search_filter = entity.SearchFilter()
+    search_filter.names = [CAR_PREFAB_FILE_NAME]
+    entities_found = len(entity.SearchBus(bus.Broadcast, 'SearchEntities', search_filter))
+    assert entities_found == 1, "Undo failed: Found duplicated entities"
+
+    general.redo()
+    wait_for_propagation()
+    entities_found = len(entity.SearchBus(bus.Broadcast, 'SearchEntities', search_filter))
+    assert entities_found == 2, "Redo failed: Failed to find duplicated entities"
```

Comment:
I suggest repeating the same validation for the children of the duplicated prefab too. Aditionally, you can also check that the parent of the duplicated children is the newly duplicated container id. This validation should be shared by duplicate_prefabs function too.

Reply:
Verified the entity count of duplicated prefab children.